src/test/java/com/portatlas/DirectoryTest.java

```diff
@@ -0,0 +1,45 @@
+package com.portatlas;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.util.Arrays;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+
+public class DirectoryTest {
+
+    private Directory dir;
+
+    @Before
+    public void setUp() {
+        dir = new Directory();
+    }
+
+    @Test
+    public void testGetDefaultDir() {
+        assertEquals(System.getProperty("user.dir") + "/public/", dir.defaultDir);
+    }
+
+    @Test
+    public void testPrintFileInDir() {
+        final File folder = new File(dir.defaultDir);
+
+        assertEquals(Arrays.asList("file1", "file2", "image.gif", "image.jpeg", "image.png", "partial_content.txt", "patch-content.txt", "text-file.txt") , dir.listFilesForFolder(folder));
```

Comment:
This line is pretty long. Is there a clean way to break it into multiple lines to improve readability?